src/main.ts

```diff
@@ -54,3 +54,8 @@ export * from "./directives/mask/mask.directive";
 export * from "./time-picker/time-picker.component";
 export * from "./directives/ripple/ripple.directive";
 export * from "./directives/toggle/toggle.directive";
+
+// services
+export * from "./services/csv/csv-exporter";
```

Comment:
Those should not be exported in the main bundle because we make JSZip a must have dependency for our package. Some users will not care for the excel export functionality at all so I don't think we should force another dependency on them.